src/panels/config/helpers/ha-config-helpers.ts

```diff
@@ -226,20 +226,19 @@ export class HaConfigHelpers extends SubscribeMixin(LitElement) {
         };
       });
 
-      if (!Object.keys(configEntriesCopy).length) {
-        return states;
-      }
-
-      const entries = Object.values(configEntriesCopy).map((configEntry) => ({
-        id: configEntry.entry_id,
-        entity_id: "",
-        icon: mdiAlertCircle,
-        name: configEntry.title || "",
-        editable: true,
-        type: configEntry.domain,
-        configEntry,
-        entity: undefined,
-      }));
+      const entries =
+        Object.keys(configEntriesCopy).length !== 0
```

Comment:
No need to check for the length because `Object.values({})` will return `[]`, right?

Reply:
Yeah, just checked and adjusted the code.